src/qcodeedit/lib/qeditor.cpp

```diff
@@ -3213,9 +3214,12 @@ void QEditor::paintEvent(QPaintEvent */*e*/)
     const QRectF rect(0, 0, width, height);
     p.fillRect(rect, bg);
 
-	p.save();
-	m_doc->draw(&p, ctx);
-	p.restore();
+	p.save();
+	m_doc->draw(&p, ctx);
+	p.restore();
+	
+	// Draw indent guides after the text is rendered
+	drawIndentGuides(&p, ctx);

```

Comment:
the indent guides must be drawn inside m_doc->draw
only indented text lines have indentation lines in the first place
unindented lines do not